smalltalksrc/Slang-Tests/SLNormalizeVariablesTestStructClass.class.st

```diff
@@ -0,0 +1,6 @@
+Class {
+	#name : 'SLNormalizeVariablesTestStructClass',
+	#superclass : 'SlangStructType',
+	#category : 'Slang-Tests',
+	#package : 'Slang-Tests'
+}
```

Comment:
Hmm this class is empty, is it needed? If needed, it would be nice to add a comment with its purpose